packages/file-upload/src/FileUpload.tsx

```diff
@@ -244,35 +233,16 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
 									Select {filesPlural}
 								</Button>
 							</Flex>
-							{value.length || fileRejections.length ? (
+							{[...value, ...fileRejections].length ? (
```

Comment:
Can we go back to what we had here? Creating an array on render can sometimes be expensive.